spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -63,6 +63,49 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
   var gradInput: A = Activity[A, T]()
 
   /**
+   * The scale of gradient weight and gradient bias
+   * before gradParameters being accumulated.
+   */
+  protected var scaleW: Double = 1.0
+  protected var scaleB: Double = 1.0
+
+  /**
+   * Get the scale of gradientWeight
+   */
+  def getScaleW(): Double = {
+    scaleW
+  }
+
+  /**
+   * Get the scale of gradientBias
+   */
+  def getScaleB(): Double = {
+    scaleB
+  }
+
+  /**
+   * Set the scale of gradientWeight
+   *
+   * @param w the value of the scale of gradientWeight
+   * @return this
+   */
+  def setScaleW(w: Double): this.type = {
```

Comment:
should we override this in container? So user can easily set a global scale @qiuxin2012